src/js/models/v2/covid19/BaseOverview.js

```diff
@@ -10,11 +10,11 @@ const BaseOverview = {
         data.funding.forEach((code) => {
             this[`_defCode_${code.def_code}_funding`] = code.amount;
         });
-        this._totalBudgetAuthority = data.total_budget_authority || null;
+        this._totalBudgetAuthority = data.total_budget_authority || null + (data.additional ? data.additional.total_budget_authority : 0);
```

Comment:
This logic should apply _only_ to the bar chart visualization, not the summary insights throughout the page. I believe we're currently referencing these 3 fields for both the chart and the summary insights. We could create 3 new fields to be used by the bar chart.

Reply:
gotcha